react/features/large-video/actions.js

```diff
@@ -21,22 +26,18 @@ export function selectParticipant() {
 
         if (conference) {
             const largeVideo = state['features/large-video'];
-            const tracks = state['features/base/tracks'];
-
             const id = largeVideo.participantId;
-            const videoTrack
-                = getTrackByMediaTypeAndParticipant(
-                    tracks,
-                    MEDIA_TYPE.VIDEO,
-                    id);
 
             try {
-                conference.selectParticipant(
-                    videoTrack && videoTrack.videoType === VIDEO_TYPE.CAMERA
-                        ? id
-                        : null);
+                conference.selectParticipant(id);
             } catch (err) {
                 _handleParticipantError(err);
+
+                sendAnalytics(createSelectParticipantFailedEvent(err));
+
+                if (typeof APP === 'object' && window.onerror) {
+                    window.onerror('', null, null, null, err);
```

Comment:
Non-blocking recommendation, maybe it'd be useful to pass a descriptive message to the window.onerror function call.

Reply:
Don't we have a helper for this?

Reply:
I couldn't find any, but a grep reveals at least one more location in meet with a similar structure. If we have a helper I agree we should use it.

Reply:
There is an unused helper defined in web outside of react. After this PR gets merged I'll try replacing the 2 or 3 places that use this format to use the helper and maybe relocate the helper.